types/spectest/tests/ssz/withdrawals_marshaling_validation.go

```diff
@@ -0,0 +1,24 @@
+package ssz
+
+import (
+	"github.com/attestantio/go-eth2-client/spec"
```

Comment:
That's one solution.

Another option is to do what Oleg wrote [here](https://github.com/bloxapp/ssv-spec/pull/234#discussion_r1195452213).

This means we can also copy paste a block from `go-eth2-client` instead of building it [ourself with `ssz.Withdarawals`](https://github.com/bloxapp/ssv-spec/blob/72023910a9d79d1151e4450668a7840646a99261/types/testingutils/beacon_node_versioned.go#L264-L267). Since we rely on `go-eth2-client` anyhow it is a trusted source.

Once we do that we can delete this test!
We wanted to do it in #234 but weren't able to reach consensus with you.

Reply:
see #258